morecantile/models.py

```diff
@@ -277,13 +339,16 @@ def custom(
 
         tms: Dict[str, Any] = {
             "title": title,
-            "identifier": identifier,
-            "supportedCRS": crs,
-            "tileMatrix": [],
+            "id": id,
+            "crs": crs,
+            "tileMatrices": [],
             "_geographic_crs": geographic_crs,
         }
 
-        is_inverted = crs_axis_inverted(crs)
+        if ordered_axes:
+            is_inverted = ordered_axis_inverted(ordered_axes)
+        else:
+            is_inverted = crs_axis_inverted(crs)
```

Comment:
@dchirst should we set `orderedAxes` in the TMS document based on the CRS information? 

To me it feels weird that one could pass a `ordered_axes` or ["Y", "X") with a CRS that would be `["X", "Y"]` ?